tests/testthat/_snaps/parsnip-survival-standalone.md

```diff
@@ -4,21 +4,21 @@
       parsnip:::.is_surv(1)
     Condition
       Error:
-      ! The object does not have class `Surv`.
+      ! The object does not have class <Surv>.
 
 # .check_cens_type()
 
     Code
       parsnip:::.check_cens_type(left_c, type = "right", fail = TRUE)
     Condition
       Error:
-      ! For this usage, the allowed censoring type is: 'right'
+      ! For this usage, the allowed censoring type is right.
```

Comment:
Ah, these need a `.val` or similar to make clear they're argument values.

Reply:
Can you change that in parsnip and I'll do updates here?

Reply:
Done!

Reply:
Danke